ReactiveUI.AndroidSupport/ReactiveAppCompatActivity.cs

```diff
@@ -0,0 +1,170 @@
+        }
+
+        public event PropertyChangedEventHandler PropertyChanged
+        {
+            add { PropertyChangedEventManager.AddHandler(this, value); }
+            remove { PropertyChangedEventManager.RemoveHandler(this, value); }
+        }
+
+        void IReactiveObject.RaisePropertyChanged(PropertyChangedEventArgs args)
+        {
+            PropertyChangedEventManager.DeliverEvent(this, args);
+        }
+
+        /// <summary>
+        /// Represents an Observable that fires *before* a property is about to
+        /// be changed.         
+        /// </summary>
+        public IObservable<IReactivePropertyChangedEventArgs<ReactiveAppCompatActivity>> Changing
+        {
+            get { return this.getChangingObservable(); }
+        }
+
+        /// <summary>
+        /// Represents an Observable that fires *after* a property has changed.
+        /// </summary>
+        public IObservable<IReactivePropertyChangedEventArgs<ReactiveAppCompatActivity>> Changed
+        {
+            get { return this.getChangedObservable(); }
+        }
+
+        /// <summary>
+        /// When this method is called, an object will not fire change
+        /// notifications (neither traditional nor Observable notifications)
+        /// until the return value is disposed.
+        /// </summary>
+        /// <returns>An object that, when disposed, reenables change
+        /// notifications.</returns>
+        public IDisposable SuppressChangeNotifications()
+        {
+            return this.suppressChangeNotifications();
+        }
+
+        public IObservable<Exception> ThrownExceptions { get { return this.getThrownExceptionsObservable(); } }
+
+        readonly Subject<Unit> activated = new Subject<Unit>();
+        public IObservable<Unit> Activated { get { return activated.AsObservable(); } }
+
+        readonly Subject<Unit> deactivated = new Subject<Unit>();
+        public IObservable<Unit> Deactivated { get { return deactivated.AsObservable(); } }
+
+        protected override void OnPause()
+        {
+            base.OnPause();
+            deactivated.OnNext(Unit.Default);
+        }
+
+        protected override void OnResume()
+        {
+            base.OnResume();
+            activated.OnNext(Unit.Default);
+        }
+
+        readonly Subject<Tuple<int, Result, Intent>> activityResult = new Subject<Tuple<int, Result, Intent>>();
+        public IObservable<Tuple<int, Result, Intent>> ActivityResult
+        {
+            get { return activityResult.AsObservable(); }
+        }
+
+        protected override void OnActivityResult(int requestCode, Result resultCode, Intent data)
+        {
+            base.OnActivityResult(requestCode, resultCode, data);
+            activityResult.OnNext(Tuple.Create(requestCode, resultCode, data));
+        }
+
+        public Task<Tuple<Result, Intent>> StartActivityForResultAsync(Intent intent, int requestCode)
```

Comment:
I realise you've probably just copied the code from the non-support implementation, but would be interested to know why we're using TPL here (and below) instead of Rx.